sbi/samplers/rejection/rejection.py

```diff
@@ -219,6 +219,8 @@ def accept_reject_sample(
             rejected. Must take a batch of parameters and return a boolean tensor which
```

Comment:
the docstring above is not correct, should be `proposal` no? Can you please correct this and add that this is a callable now that takes a sample shape as arg and additional kwargs?

Reply:
Nice catch, yes I will update this!